### PR TITLE
Update index.adoc

### PR DESCRIPTION
**Pull request by voroyam**
Add links to this empty page to give it some meaning.

I added links to our guides on how to install ownCloud with some description. This way a user can easily access the main parts of the installation section.

needs to be back ported to 10.5, 10.4, 10.3.

### CODE REVIEW

File: modules/admin_manual/pages/installation/index.adoc [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
 
+== xref:installation/manual_installation.adoc[Manual Installation]
+
+This is a thorough guide to installing ownCloud, containing all the information needed for the prerequisites, the dependencies, the actual installation and the configuration afterwards.
+
+== xref:installation/docker/index.adoc[Installing With Docker]
+
+This guide will show you how to install ownCloud with Docker Compose using our YAML file.
+
+== xref:installation/ubuntu_18_04.adoc[Example Installation on Ubuntu]
```
Comment (cdamken) [inline modules/admin_manual/pages/installation/index.adoc:14]: Could we change 18.04 to 20.04? I think the commands are the same with exception of the php-smbclient.

**Review by cdamken — APPROVED** [review]
I find this a great idea. works for me ;)

File: modules/admin_manual/pages/installation/index.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/installation/index.adoc:14]: yes, but then we also need to add a note about the Ondrej repo.

File: modules/admin_manual/pages/installation/index.adoc [hunk 2]
```diff
@@ -1,5 +1,20 @@
 = Installation
 :page-aliases: go/admin-install.adoc
 
-In this section, you will find all the information you need for installing ownCloud. 
+You can install ownCloud in multiple ways, here are our trusted guides:
 
+== xref:installation/manual_installation.adoc[Manual Installation]
```
Comment (mmattel) [inline modules/admin_manual/pages/installation/index.adoc:6]: You can not have a reference in parallel being a section. You must have a fixed text for the section and in a new independent line, you can do the reference with xref.

Comment (voroyam) [reply modules/admin_manual/pages/installation/index.adoc:6]: You can not or you should not? In my local doc installation it works.

**Comment by mmattel** [comment]
Fixed paths and names.
Tested with `yarn run antora`

**Comment by mmattel** [comment]
Note: backport can only be done for 10.6 and 10.5

**Pull request merged**